doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -604,19 +537,19 @@ type of each branch can depend on the type dependencies specific to the
 branch and the whole pattern matching expression has a type determined
 by the specific dependencies in the type of the term being matched. This
 dependency of the return type in the annotations of the inductive type
-is expressed using a “:g:`in` :math:`I` :g:`_ … _` :token:`pattern`:math:`_1` …
-:token:`pattern`:math:`_n`” clause, where
+is expressed with a clause in the form
+:n:`in` :math:`I` :n:`_ … _ @pattern … @pattern`, where
```

Comment:
Let's stop mixing math-style and notation-style, that makes no sense. Furthermore, let's avoid ellipses when we can use notations for repetition:

```suggestion
:n:`in @qualid {+ _ } {+ @pattern }`, where
```